src/Editor/CodelensProvider.ts

```diff
@@ -35,7 +36,16 @@ export class CodelensProvider implements vscode.CodeLensProvider {
         let range = document.getWordRangeAtPosition(position, /(.+)/g) as vscode.Range;
         let lineStr = line.text;
 
-        // TODO Add more
+        if (lineStr.indexOf('=') === -1) {
+          toolsAttr.forEach((tool) => {
+            if (tool.name === lineStr) {
+              codeLenses.push(new vscode.CodeLens(range));
```

Comment:
what if `range` is null or undefined? does `CodeLens` work as expected?

Reply:
If `range` is undefined, an error will occur.
As far as I know, undefined is returned if someone simply typed 'Enter'. 
However, `while((matches==regex.exe(activeEditorText))!==null)` primarily checks for which characters are present.
So I didn't add `if(range)`.